include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -417,6 +444,43 @@ struct RunOptions : detail::Base<OrtRunOptions> {
   RunOptions& UnsetTerminate();
 };
 
+
+namespace detail {
+// Utility function that returns a SessionOption config entry key for a specific custom operator.
+// Ex: custom_op.[custom_op_name].[config]
+std::string MakeCustomOpConfigEntryKey(const char* custom_op_name, const char* config);
+}  // namespace detail
+
+/// <summary>
+/// Class that represents session configuration entries for one or more custom operators.
+///
+/// Example:
+///   Ort::CustomOpConfigs op_configs;
+///   op_configs.AddConfig("my_custom_op", "device_type", "CPU");
+///
+/// Passed to Ort::SessionOptions::RegisterCustomOpLibrary.
+/// </summary>
+struct CustomOpConfigs {
+  CustomOpConfigs() = default;
+  ~CustomOpConfigs() = default;
+  CustomOpConfigs(const CustomOpConfigs&) = default;
+  CustomOpConfigs& operator=(const CustomOpConfigs&) = default;
+  CustomOpConfigs(CustomOpConfigs&& o) = default;
+  CustomOpConfigs& operator=(CustomOpConfigs&& o) = default;
+
+  // Add a configuration entry for a specific custom operator. Returns a reference to *this to allow chaining calls.
+  CustomOpConfigs& AddConfig(const char* custom_op_name, const char* config_key, const char* config_value);
```

Comment:
>const char* custom_op_name [](http://example.com/codeflow?start=29&length=26)

We will need to add documentation for each parameter. let's explain that custom_op_name is needed in cases the library implements multiple custom ops.

Reply:
Added a more detailed description of the method and its parameters